libs/build_processed_dataset.py

```diff
@@ -175,18 +169,26 @@ def get_usa_by_county_with_projection_df(input_dir, intervention_type):
     return counties
 
 
-def get_usa_by_states_df(input_dir, intervention_type):
+def get_usa_by_states_df(input_dir: str, intervention_type):
```

Comment:
is intervention_type here a string too? or `Intervention`

Reply:
fixed